concepts/05 Widgets/Switch/10 Handle the Value Change Event.md

```diff
@@ -11,6 +14,96 @@ To process a new **Switch** value, you need to handle the value change event. If
         });
     });
 
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-{widget-name} (onValueChanged)="switchValueChanged($event)">
```

Comment:
It looks like in Angular and Vue we should show two-way binding and in React state changing.

Reply:
[please notice](https://github.com/DevExpress/devextreme-documentation/pull/331#discussion_r380236985)